entries/p/platform360.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Plesk 360": {
+    "domain": "platform360.io",
```

Including the old domain might help some password managers.
```suggestion
    "domain": "platform360.io",
    "additional-domains": [
      "my.plesk.com"
    ],
```